guides/advanced-use/pods.md

```diff
@@ -0,0 +1,92 @@
+Ember supports two different project structures - `classic` and `pods-based`.
+
+The classic project structure organizes the filesystem by entity types.  For example, if you have a `post` resource and a `tags` component in your project, the `classic` structure would have this filesystem:
+
+```sh
+app
+├── components
+|   └── tags.js
+├── controllers
+|   └── post.js
+├── models
+|   └── post.js
+├── routes
+|   └── post.js
+└── templates
+    ├──components/tags.hbs
+    └── post.hbs
+```
+
+Pods-based projects organize files by features, combining all entity files into a common directory.  The aforementioned example as a `pods-based` project would have this filesystem:
+
+```sh
+app
+├── components
+|   └── tags
+|       ├── component.js
+|       └── template.js
+└── post      
+    ├── controller.js
+    ├── model.js
+    ├── route.js
+    └── template.hbs
+```
+To create the pods structure in an application you add the `--pod` option the the `ember generate` command.  For example, to generate the `tags` component, you would run `ember generate component tags --pod`.
+
+As your app becomes larger, a feature-driven structure may be better. Splitting your application by functionality/resource would give you more power and control to scale and maintain it.
+
+### Classic or Pods?
+<!-- Is this going to be confusing? -->
+Ember projects do not have to be either `classic` or `pods-based`, they can also be mixed projects. As a default, if a file is not found on the pod structure, the Ember Resolver will look it up within the classic filesystem structure.
+
+If you start with a `classic` project, you can switch to the `pods-based` filesystem as your project grows without changing the existing `classic` filesystem. You can migrate the `classic` structure as time allows in the future.
+
+### Pods as Default
+If you would like to use the pods structure as the default without using the `--pod` option, you can set `usePods` in `.ember-cli`:
+
+```js
+// .ember-cli
+{
+    "usePods": true
+}
+```
+
+### podModulePrefix
+Rather than hold your resource directories on the root of your app you can define a POD path using the attribute podModulePrefix within your environment configs. The POD path should use the following format: `{appname}/{poddir}`.
+
+```js
+// config/environment.js
+module.exports = function(environment) {
+  let ENV = {
+    modulePrefix: 'my-new-app',
+    // namespaced directory where resolver will look for your resource files
+    podModulePrefix: 'my-new-app/pods',
+    environment: environment,
+    rootURL: '/',
+    locationType: 'auto'
+    //...
+  };
+
+  return ENV;
+};
+```
+
+Then your directory structure would be:
+
+```sh
+app
+└── pods
+    ├── components
+    |   └── tags
+    |   ├── component.js
+    |   └── template.js
+    └── post      
+        ├── controller.js
+        ├── model.js
+        ├── route.js
+        └── template.hbs
+```
+
+### Module Unification
```

Comment:
My gut feeling is that this section about Module unification should be at the top 🤔 I know we haven't officially deprecated pods but I think we should warn people that there is a "pod-like" change coming so they may want to hold off switching to pods... I'm not sure what is best here

Reply:
At the top would be better.  I think it will be easier to write a MU blurb at the top vs at the end, if that makes sense.  I might take a stab at it this evening but anybody else is welcome to write something too :grin:

Reply:
I prefer this at the end. We want to avoid baiting devs into trying to "future proof" their apps, as well as misleading them into thinking that MU is a complete, done deal, exactly as written in the RFC. Most implementations end up a little different than the RFC itself, and this can lead to painful, costly missteps if a dev makes the wrong assumptions.

Perhaps we could call this section "common questions". The question could be "I heard about another file structure called module unification. What is that?"  "Module unification is a future-looking proposal that included a new file structure, one that is most similar to pods. To follow along with the progress of new features in Ember, check out the [Ember Blog](). If changes are made to the app file structure in the future, they will be accompanied with codemods to make migrations easier, both for classic and pod structures. Developers who want to test drive the experimental Module Unification layout should read `<link>`. In general, experimental features should be used with caution, as they are subject to breaking changes, significant iteration, and in some cases, they may be removed altogether."

I'd also want the input of a CLI or MU leader on this section.

Reply:
I have received feedback that we shouldn't use the term "Module Unification" here because it means a ton of other things too, so we should call it "new file layout" when writing about it.